### PR TITLE
Chore: Add DynamoDB alternative log location stack

### PR DESCRIPTION
**Pull request by RLRabinowitz**
Add stack for using an alternative log location - DynamoDB

### CODE REVIEW

File: alternative-log-storage/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Alternative Log Storage
```
Comment (HeverFarber) [inline alternative-log-storage/README.md]: Maybe the folder name only `log-storage`

Add short info in the main readme with reference to here

File: alternative-log-storage/aws/dynamodb/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## DynamoDB table on your own cloud provider
+
+You can apply this module on your AWS account, to create a DynamoDB table for storing logs. env0 will assume the role created in this module, in order to read and write logs
+
+#### Variables:
+
+- region - AWS region in which the dynamodb will be created. Should be provided to env0 as well
+- external_id - An external ID provided by env0 for assuming the roles to read and write from and to log table
```
Comment (HeverFarber) [inline alternative-log-storage/aws/dynamodb/README.md]: Maybe add that info to our wiki - what which we need to provide to the customer. agentKey, externalId

File: alternative-log-storage/aws/dynamodb/main.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+provider "aws" {
```
Comment (HeverFarber) [inline alternative-log-storage/aws/dynamodb/main.tf]: you should not add provider here - they need to use it as module or fork it

File: alternative-log-storage/aws/dynamodb/versions.tf [hunk 4]
```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = "1.0.0"
```
Comment (HeverFarber) [inline alternative-log-storage/aws/dynamodb/versions.tf]:
```suggestion
  required_version = ">= 1.0.0"
```

**Review by HeverFarber — COMMENTED** [review]
Nice

File: alternative-log-storage/README.md (hunk 1 above)
Comment (HeverFarber) [reply alternative-log-storage/README.md]: Add an example how to use it as module

File: alternative-log-storage/aws/dynamodb/README.md (hunk 2 above)
Comment (RLRabinowitz) [reply alternative-log-storage/aws/dynamodb/README.md]: Already updated [this wiki](https://github.com/env0/env0/wiki/Migrating-customers-to-K8S-agent), I'll link the actual module once it's merged to master

File: alternative-log-storage/README.md (hunk 1 above)
Comment (RLRabinowitz) [reply alternative-log-storage/README.md]: 4a3b78067256fa82e639444b5e6cee646c0df1f3

File: alternative-log-storage/aws/dynamodb/main.tf (hunk 3 above)
Comment (RLRabinowitz) [reply alternative-log-storage/aws/dynamodb/main.tf]: a2f22176e9aa204ce0eba5a9b77e5a7c4e38bc60

File: alternative-log-storage/aws/dynamodb/versions.tf (hunk 4 above)
Comment (RLRabinowitz) [reply alternative-log-storage/aws/dynamodb/versions.tf]: a2f22176e9aa204ce0eba5a9b77e5a7c4e38bc60

**Review by HeverFarber — APPROVED** [review]
🔥

**Comment by RLRabinowitz** [comment]
[Using custom dynamodb table - docs and migration](https://trello.com/c/7jOh8cly/5269-using-custom-dynamodb-table-docs-and-migration)

**Pull request merged**